v0/destinations/ga4/data/GA4CommonConfig.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "destKey": "client_id",
+    "sourceKeys": "messageId",
```

Comment:
lets add support for some custom id as well for this`context.client_id || messageId`